app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditViewModel.kt

```diff
@@ -23,21 +23,21 @@ class ProfileEditViewModel @Inject constructor(
 
   private val isAllowedDownloadAccessMutableLiveData = MutableLiveData<Boolean>()
 
-  /** Specifies whether download access has been enabled by the user. */
+  /** The download access for the profile. */
```

Comment:
I think the previous KDoc was a bit clearer (since it described the 'enabled' bit), but perhaps could do without the 'Specifics' bit in the beginning.